scte35/scte35.go

```diff
@@ -183,3 +230,148 @@ func abs(num int8) int8 {
 func uint40(buf []byte) gots.PTS {
 	return (gots.PTS(buf[0]&0x1) << 32) | (gots.PTS(buf[1]) << 24) | (gots.PTS(buf[2]) << 16) | (gots.PTS(buf[3]) << 8) | (gots.PTS(buf[4]))
 }
+
+// String returns a string representation of the SCTE35 message.
+// String function is for debugging and testing.
+func (s *scte35) String() string {
```

Comment:
We've already talked about this, but it might be worthwhile to output JSON instead of this crafted string. This could be addressed in another PR though.

Reply:
Or have multiple methods: one for a formatted string and another for a JSON formatted string.